kore/test/Test/Kore/Repl/Parser.hs

```diff
@@ -303,6 +311,29 @@ saveSessionTests =
     , "save-session"       `fails`     "need to supply file name"
     ]
 
+initScriptTests :: [ParserTest [ReplCommand]]
+initScriptTests =
+    [ script1 `parsesTo_`
```

Comment:
I think this would look better as
```
    let
        script = "... string "
        commands  = [ ... commands ]
    in
        script `parsesTo_` commands